apps/web/src/routes/+layout.svelte

```diff
@@ -1,52 +1,36 @@
 <script lang="ts">
-	import '../app.css';
-	import { onMount } from 'svelte';
+	import '$lib/styles/global.css';
+	import { createAuthService } from '$lib/auth/authService.svelte';
+	import Navigation from '$lib/components/Navigation.svelte';
+	import type { Snippet } from 'svelte';
+	import { goto } from '$app/navigation';
 	import { page } from '$app/stores';
-	import { env } from '$env/dynamic/public';
 
-	// onMount, check for page params
-	let token: string | null = null;
-	onMount(() => {
-		// if searchparams has token, save it to localstorage
+	interface Props {
+		children: Snippet;
+	}
+
+	const { children }: Props = $props();
+
+	const authService = createAuthService();
+
+	$effect(() => {
 		if ($page.url.searchParams.get('gb_access_token')) {
-			let token = $page.url.searchParams.get('gb_access_token');
+			const token = $page.url.searchParams.get('gb_access_token');
 			if (token && token.length > 0) {
-				localStorage.setItem('gb_access_token', token);
-				// redirect to remove search params
-				window.location.href = '/';
+				authService.setToken(token);
```

Comment:
Yeah definitely a fair point. 

If we want to persist the token though there's also the argument for using cookies instead of localstorage because of their httpOnly flag so they can't be targeted from JS. 

Right now I just cleaned up what was already there and didn't really change any fundamental functionality. I think this is very much still in the experimental/prototype stage. 

But this is definitely a conversation worth having. I'll bring it up with the folks in the office this week too 🙏

Reply:
Also sidenote, this is the web app, it's not included in any "release" per se, right. 

And we had already stored it in localstorage previous to this PR, like I said I just moved stuff around a bit